html-web-page/index.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>HTML Web Page</title>
+  <style>
+    table,
+    th,
+    td {
+      border: 1px solid black;
+    }
+
+    td:not(:first-child) {
+      text-align: center;
+    }
+  </style>
+</head>
+<body>
+  <h1>Jelly of the Month Club</h1>
+  <h3>Make all your friends jelly of your stock of jelly!</h3>
+  <div>
+    Here at JOTMC Inc., we are constantly striving to supply our customers with the highest-quality and the most unique jellies on a monthly basis. For the jelly lover in your life (or yourself!), this is the gift that keeps on giving all year round. We offer three tirs of monthly jelly joy, and each one offers their own perks. Here is a breakdown of each, as well as your monthly price:
+  </div>
+  <table>
+    <thead>
+      <tr>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td>Three new curated Jellies per month</td>
+        <td>&check;</td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly jellies come with commemorative mason jars</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free bonus jar of preserves bi-monthly</td>
+        <td></td>
+        <td>&check;</td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Free yearly tickets to Jelly Con</td>
+        <td></td>
+        <td></td>
+        <td>&check;</td>
+      </tr>
+      <tr>
+        <td>Monthly Pricing</td>
+        <td>$30</td>
+        <td>$50</td>
+        <td>$60</td>
+      </tr>
+    </tbody>
+  </table>
+  <form>
+    <div><label for="name">Like what you see? Become a part of the JOTMC family today!</label></div>
+    <div>
+      <input type="text" value="Your name" id="name">
+      <label for="Plan">
+        <select>
+          <option>Select Your Plan</option>
+          <option>Basic</option>
+          <option>Deluxe</option>
+          <option>JELLY GIANT</option>
+        </select>
+      </label>
```

Comment:
You don't need to use the `for` attribute here. It's either use the `for` attribute, or wrap the element inside a `label`. 

This actually doesn't even need a `label` in general, because there's no text associated with it.